### ISSUE TITLE
Build and send data quality notification emails

### ISSUE THREAD

**Issue by CharliePatterson** [digital-land/submit#427]
## Context:

- The data providers dashboard surfaces data quality issues to LPAs, to help them understand how to improve the quality of the data they've submitted to the platform.
- However, unless an LPA is regularly checking their dashboard, they won't know that there are tasks for them to complete.
- As such, there's a need for the platform to notify LPAs of these things, and signpost them to the relevant pages to complete these actions.
- The proposed plan of action is here: https://docs.google.com/document/d/1eFYaBbhmIP5IOnZ4nYGeaGIu0Mq16xCcHVJ-Kxg3ZAU/edit

## Acceptance critera:

The scope of this ticket covers the work to build and send the emails.

## Tasks:

- [x] Secure emails via community management (Tom + Frankie)
- [x] Draft service messages for approval 
- [x] Group LPAs into A/B test
- [x] Find UTM builder and test it works
- [x] Review and secure approval for Service Messages from Charlie 
- [x] Content review with Gwen
- [x] Conduct test of service emails
- [x] Review findings of testing for service emails
- [x] Playback findings to the team and suggest long-term plan

## Dates of email campaign
- [x] 16/09/2024
- [ ] ~~24/09/2024~~
- [x] 02/10/2024
- [x] 10/10/2024 
- [ ] 15/10/2024 - Final

**Comment by CharliePatterson** [comment]
Copying @MJFaucherFolie's comment from another ticket:

Drafted Service Messages for review:

- General message: https://docs.google.com/document/d/1YnrulDYSZMmE9iZoA3hMwgtYByT2br64Ye7U7-9eTYk/edit 
- Descriptive message: https://docs.google.com/document/d/17o3N-5h_QxVt-yCi3qfDum-RJZrkzA-p0Xvl10zP9jg/edit

Will update once Charlie has had a review and shared feedback

**Comment by MJFaucherFolie** [comment]
Gwen has reviewed and approved
Sent test email to Charlie 
Sent to LPAs in sample size
Will review Smartlook in near future to measure impact before launch of send set of emails

**Comment by MJFaucherFolie** [comment]
Began a template to track impact of service messages: https://docs.google.com/spreadsheets/d/1MlbQ6Ghl_i_ISuF9lR58MOG4JLNF4qQMdJfLcFkRJ0A/edit?gid=182178139#gid=182178139

**Comment by MJFaucherFolie** [comment]
Second batch of emails drafted and ready to launch. Waiting on the nav ticket #420 before launching, so users can explore issues

**Comment by MJFaucherFolie** [comment]
Completed test and shared results in an SLT meeting. Positive feedback and encouragement to continue.

Next steps - Widen the sample size, iterate content and focus on automating the process

**Comment by maddie-broxup** [comment]
Have also assigned myself to this as I'm picking up some possible iterations with MJ tomorrow

**Comment by MJFaucherFolie** [comment]
> Have also assigned myself to this as I'm picking up some possible iterations with MJ tomorrow

Retiring this ticket and assigning u8 to a new one dedicated to [Improving the content of data quality emails #578](https://github.com/digital-land/submit/issues/578)
